### ISSUE TITLE
Add comparison page

### ISSUE THREAD

**Issue by davelab6** [impallari/Font-Testing-Page#70]
@impallari will you accept a PR that adds a comparison page similar to http://sf-playground.github.io/hackover/ ? :)

**Comment by davelab6** [comment]
So, you won't accept such a PR if someone else does it?

**Comment by impallari** [comment]
Yes, I will, but it has to be done in a way that it's somewhat simple and encapsulated, so it don't mess around with the rest of the page

**Comment by davelab6** [comment]
Sure! Please re-open :)
